style.css

```diff
@@ -2440,6 +2440,21 @@ ul {
   text-transform: lowercase;
 }
 
+/***** Community Badges *****/
+/* Styles labels used next to the authors of article comments, community posts, and community comments */
+.community-badge {
```

Comment:
Is there a reason that this is duplicated both here and in the `_community-badge.css` file?

Reply:
"style.css" is generated from the styles/*.css files :-)

Reply:
Aaah, got it :D